api/models/index.js

```diff
@@ -77,3 +77,16 @@ exports.newComment = (review_id, author, body) => {
       return res.rows[0];
     });
 };
+exports.increaseVote = (review_id, votes) => {
```

Comment:
careful with naming here too - same as before it could one day patch more than just the votes - and it could increase or decrease - keep naming nice and generic